provider/keycloak_openid_client_authorization_permission.go

```diff
@@ -0,0 +1,177 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"strings"
+)
+
+var (
+	keycloakOpenidClientResourcePermissionDecisionStrategies = []string{"UNANIMOUS", "AFFIRMATIVE", "CONSENSUS"}
+	keycloakOpenidClientPermissionTypes                      = []string{"resource", "scope"}
+)
+
+func resourceKeycloakOpenidClientAuthorizationPermission() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationPermissionCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationPermissionRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationPermissionDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationPermissionUpdate,
+		// This resource can be imported using {{realm}}/{{resource_server_id}}. The Client ID is displayed in the GUI
```

Comment:
This comment doesn't seem correct.  You can just remove it if you want.

Reply:
Removed